types/pg-cursor/tsconfig.json

```diff
@@ -14,10 +14,11 @@
         ],
         "types": [],
         "noEmit": true,
+        "esModuleInterop": true,
```

Comment:
this is expected to be set on consuming code. By default this is set in generated `tsconfig.json` from some time.
DT is not the best place to do this, what's the use case?
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports